src/Request.php

```diff
@@ -25,13 +25,11 @@ class Request implements RequestInterface
     private $uri;
 
     /**
-     * @param null|string $method HTTP method for the request.
```

Comment:
method as null will not fail because it uses `strtoupper` on it which transforms it to a string. But it should not be a documented way because it's not consistent with `getMethod` and `withMethod`